BREAKING.md

```diff
@@ -16,6 +16,7 @@ There are a few steps you can take to write a good change note and avoid needing
 - [Deprecated `Loader._create` is removed](#deprecated-loadercreate-is-removed)
 - [Stop exporting internal class `CollabWindowTracker` ](#stop-exporting-internal-class-collabwindowtracker)
 - [base-host package removed](#base-host-package-removed)
+- [Registers removed from sequence and merge-tree](#Registers-removed-from-sequence-and-merge-tree)
```

Comment:
Can you please add a note to docs/content/docs/updates/v0.51.md. Just a note with the PR number or something. I curate the notes themselves but I am encouraging people to note things there to help me when I write release notes.

While I understand that this stuff should have been impossivble to use, since it is part of the public API surface I want to include it in the notes.

Reply:
Added in latest